index.html

```diff
@@ -4,9 +4,15 @@
 redirect_from:
   - about.html
 ---
+        <!-- begin banner announcement -->
+        <section class="announcement bg-idr-purple">
+            <span><i class="fa fa-bullhorn"></i> Please note that data download via Aspera is currently unavailable. <a class="styled-link" href="https://www.openmicroscopy.org">Click link to read update</a></span>
```

Comment:
```suggestion
            <span><i class="fa fa-bullhorn"></i>IDR storage and compute services are currently being migrated. This may lead to occasional service outages. Please contact <a class="styled-link" href="mailto:idr@openmicroscopy.org">the IDR team</a> if you experience any problems.</span>
```